drivers/gles3/storage/light_storage.h

```diff
@@ -33,6 +33,10 @@
 
 #ifdef GLES3_ENABLED
 
+// This must come first to avoid windows.h mess
```

Comment:
```suggestion
// These must come first to avoid windows.h mess.
```

Reply:
Though this comment calling out windows.h is getting old (even though I probably wrote it). Would be worth checking whether it's actually a problem, if not we can get rid of the comments and the forced order.

Reply:
> Though this comment calling out windows.h is getting old (even though I probably wrote it).

It's not (at least with MinGW), order is still important, or it's gonna have a lot of conflicts in `Object`.